### PR TITLE
Rebase to 3.17, migrate to s6v3, switch to nginx

### PR DESCRIPTION
**Pull request by thespad**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-smokeping/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

File: root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+fi
+
+if [[ ! -f /usr/bin/tcpping ]]; then
+    install -m755 -D /defaults/tcpping /usr/bin/
+fi
+
+# permissions
+chmod 777 /var/cache/fontconfig/
+
+chown -R abc:abc \
```
Comment (nemchik) [inline root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run:39]: `lsiown` ?

Comment (thespad) [reply root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run]: Fixed

**Review by nemchik — APPROVED** [review]

**Comment by VictorLopess** [comment]
Guys. Is it correct? Why maintain the same tag (2.7.3) in a big change like that?

**Comment by aptalca** [comment]
2.7.3 is not our tag, it's the upstream version

**Comment by nemchik** [comment]
> Guys. Is it correct? Why maintain the same tag (2.7.3) in a big change like that?

Our images are tagged with the version of the upstream project (smokeping version 2.7.3) and our own build number at the end.

For example:
https://github.com/linuxserver/docker-smokeping/releases/tag/2.7.3-r9-ls14
And
https://github.com/linuxserver/docker-smokeping/releases/tag/2.7.3-r5-ls13

The `r9` vs `r5` comes from the Alpine package revision:
https://pkgs.alpinelinux.org/package/v3.17/main/x86_64/smokeping
And
https://pkgs.alpinelinux.org/package/v3.13/main/x86_64/smokeping

Anyway, the important thing is the build number `ls13` vs `ls14` in this case. We don't do semver, just increase build counter.

**Comment by thespad** [comment]
Unfortunately Alpine haven't updated the smokeping version in years, it's the same in 3.13 as it is in Edge, so we didn't have any other option.

**Pull request merged**